src/main/java/nextstep/auth/authentication/TokenAuthenticationInterceptor.java

```diff
@@ -39,15 +48,32 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
     }
 
     public AuthenticationToken convert(HttpServletRequest request) throws IOException {
-        // TODO: request에서 AuthenticationToken 객체 생성하기
-        String principal = "";
-        String credentials = "";
+        String requestInfo = request.getReader()
+                .lines()
+                .findFirst()
+                .orElseThrow(RuntimeException::new);
+
+        JSONObject jsonObject = new JSONObject(requestInfo);
+
+        String principal = jsonObject.get("email").toString();
+        String credentials = jsonObject.get("password").toString();
```

Comment:
`jsonObject`는 특정 타입의 값을 가져오는 API 제공합니다 ㅎㅎ

```suggestion
        String principal = jsonObject.getString("email");
        String credentials = jsonObject.getString("password");
```